src/java/org/apache/cassandra/db/compaction/unified/StaticController.java

```diff
@@ -150,6 +151,10 @@ static Controller fromOptions(Environment env,
         {
             logger.warn("Unable to parse saved flush size. Using starting value instead:", e);
         }
+        catch (FSError e)
```

Comment:
Should we just catch an even more generic thing like RuntimeException or Exception?  Any error reading the data we use the default.

Reply:
sure, added.